src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -284,7 +290,7 @@ def remove_subvolume_snapshot(self, volname, subvolname, snapname, groupname, fo
 
     ### group operations
 
-    def create_subvolume_group(self, volname, groupname):
+    def create_subvolume_group(self, volname, groupname, mode=None):
```

Comment:
What about setting a pool on a subvolume group too? That seems very useful. Add a test too.

Reply:
Done